pkg/sdk/row_access_policies_gen_test.go

```diff
@@ -49,7 +53,7 @@ func TestRowAccessPolicies_Create(t *testing.T) {
 
 	t.Run("one parameter", func(t *testing.T) {
 		opts := defaultOpts()
-		assertOptsValidAndSQLEquals(t, opts, "CREATE ROW ACCESS POLICY %s AS (n VARCHAR) RETURNS BOOLEAN -> true", id.FullyQualifiedName())
+		assertOptsValidAndSQLEquals(t, opts, `CREATE ROW ACCESS POLICY %s AS ("n" VARCHAR) RETURNS BOOLEAN -> true`, id.FullyQualifiedName())
```

Comment:
That also breaking change that could be mentioned in the migration guide. Now, we make arguments case sensitive, so it's different on Snowflake side (they have to reference them with quotes). Btw that doesn't need a state upgrader that would convert names to uppercase in the newer version ?

Reply:
It's mentioned in the migration guide. Adjusted the state upgrader and test.